### ISSUE TITLE
Filter nested paginate queries

### ISSUE THREAD

**Issue by jgb-solutions** [nuwave/lighthouse#1058]
Hey guys I don't know if that feature exists already but I couldn't seem to find it in the docs.
Consider this user type:
```
type User {
	id: ID!
	name: String!
	email: String
	avatar: String
	active: Boolean!
	tracks: [Track!]! @hasMany(type: "paginator", defaultCount: 20)
	artists: [Artist!]! @hasMany(type: "paginator", defaultCount: 20)
	telephone: String
	created_at: DateTime!
	updated_at: DateTime!
}
```
I want to be be able to order the artists relationship by a column name like:
```
type User {
	id: ID!
	name: String!
	email: String
	avatar: String
	active: Boolean!
	tracks: [Track!]! @hasMany(type: "paginator", defaultCount: 20)
	artists: [Artist!]!
		@hasMany(type: "paginator", defaultCount: 20)
		@orderBy(field: "created_at", order: ASC)
	telephone: String
	created_at: DateTime!
	updated_at: DateTime!
}
```

I use the @orderBy directive fine when passed as parameters like:
```
	genres(
		page: Int = 1
		orderBy: [OrderByClause!] = [{ field: "name", order: ASC }] @orderBy
	): [Genre!]! @paginate(defaultCount: 20)
```

but I can't get it to work for relationships. Does the package already have this feature or not yet? 

Thanks for this package!

**Comment by jgb-solutions** [comment]
I solved it by using a different hasMany relation on the User class like this:
<img width="687" alt="Screen Shot 2019-11-26 at 3 20 36 PM" src="https://user-images.githubusercontent.com/8051752/69669704-5bb95980-1060-11ea-883e-32185988b5f2.png">

calling the name is the graphql schema like this:
<img width="648" alt="Screen Shot 2019-11-26 at 3 20 15 PM" src="https://user-images.githubusercontent.com/8051752/69669732-6e339300-1060-11ea-9751-e925911a230f.png">

**Comment by jgb-solutions** [comment]
By reading the docs again I found the `scopes` option on `@hasMany` that is an array that takes a scope function names on your model. So I could've solved it with `@hasMany(scopes: ["byStageName"]) and one. But you would also need the scope on your model like so: 
```
public function scopebyName($query)
{
   $query->orderBy('name');
}
```

Hope that will help someone. I don't why but it seems the issues around here don't have many responses. Anyway I solved mine thanks to the docs.
